### DISCUSSION TITLE
Restore MUGEN 1.0 parallax

### DISCUSSION THREAD

**Discussion by Ksylatarsh** [ikemen-engine/Ikemen-GO#1044] [Feature Request]
### Is your feature request related to a problem? Please describe.

The old 1.0 parallax actually looked way better than the newer one, i don't know if it's something related to the filters or something but it looked less blurry and less "broken", I don't know the term exactly for this but I'll attach some screenshots of 1.0 parallaxed floor vs. 1.1/IKEMEN parallaxed floor. Used the same stage for better comparison.

### Describe the solution you'd like

Solution is simple, revert back to 1.0 parallax or sprite filters.

### Additional context

![mugen000](https://user-images.githubusercontent.com/126763763/224534191-bd9324b9-772c-47b3-b03c-748c76f0f34a.png)
MUGEN 1.0

![ikemen001](https://user-images.githubusercontent.com/126763763/224534217-84717220-4f14-4acf-98a5-49c72d203484.png)
MUGEN 1.1/IKEMEN

**Comment by potsmugen** [comment]
I've noticed the same. The old parallax looks a lot better for the low resolution stages. I think this could be a toggle in the Parallax controller. The default value could vary according to the Mugenversion specified for the stage.

**Comment by Orochikyocr** [comment]
Just FYI, it is not like the devs decided to do the parallax differently than MUGEN. Since MUGEN is a closed source the reason for these differences is that nobody knows how parallax works on Mugen.
